src/s1reader/s1_burst_slc.py

```diff
@@ -368,3 +368,62 @@ class represents a polynomial function of range
                                   rg_order)
 
         return az_carrier_poly
+
+    def as_dict(self):
+        """
+        Return SLC class attributes as dict
+
+        Returns
+        -------
+        self_as_dict: dict
+           Dict representation as a dict
+        """
+        self_as_dict = {}
+        for key, val in self.__dict__.items():
+            if key == 'sensing_start':
+                val = str(val)
+            elif key == 'center':
+                val = val.coords[0]
+            elif isinstance(val, np.float64):
+                val = float(val)
+            elif key == 'azimuth_fm_rate':
+                temp = {}
+                temp['order'] = val.order
+                temp['mean'] = val.mean
+                temp['std'] = val.std
+                temp['coeffs'] = val.coeffs
+                val = temp
+            elif key == 'border':
+                val = self.border[0].wkt
+            elif key == 'doppler':
+                temp = {}
+
+                temp['poly1d'] = {}
+                temp['poly1d']['order'] = val.poly1d.order
+                temp['poly1d']['mean'] = val.poly1d.mean
+                temp['poly1d']['std'] = val.poly1d.std
+                temp['poly1d']['coeffs'] = val.poly1d.coeffs
+
+                temp['lut2d'] = {}
+                temp['lut2d']['x_start'] = val.lut2d.x_start
+                temp['lut2d']['x_spacing'] = val.lut2d.x_spacing
+                temp['lut2d']['y_start'] = val.lut2d.y_start
+                temp['lut2d']['y_spacing'] = val.lut2d.y_spacing
+                temp['lut2d']['length'] = val.lut2d.length
+                temp['lut2d']['width'] = val.lut2d.width
+                temp['lut2d']['data'] = val.lut2d.data.flatten().tolist()
+
+                val = temp
+            elif key in ['orbit']:
+                temp = {}
+                temp['ref_epoch'] = str(val.reference_epoch)
+                temp['time'] = {}
+                temp['time']['first'] = val.time.first
+                temp['time']['spacing'] = val.time.spacing
+                temp['time']['last'] = val.time.last
+                temp['time']['size'] = val.time.size
+                temp['position'] = val.position.flatten().tolist()
```

Comment:
Position is 2 dimensional with a shape of number_of_state_vectorx by 3. The three columns of the 2D array represent the coordinates in X, Y and Z directions. To avoid any confusion in reconstructing orbit from the stored metadata, I suggest that we be very clear on the three components: 
```suggestion
                temp['positionX'] = val.position[:,0].tolist()
                temp['positionY'] = val.position[:,1].tolist()
                temp['positionZ'] = val.position[:,2].tolist()
```